SalesforceSDK/Salesforce.Sample.HybridMainPage.Universal/www/js/index.js

```diff
@@ -0,0 +1,114 @@
+var oauth = new SalesforceJS.OAuth2();
```

Comment:
I see that there is a default.js file too. SalesforceSDK/Salesforce.Sample.HybridMainPage.Universal/www/js/default.js